docs/reference/cat/alias.asciidoc

```diff
@@ -4,6 +4,14 @@
 <titleabbrev>cat aliases</titleabbrev>
 ++++
 
+[IMPORTANT]
+====
+cat APIs are only intended for human consumption using the
+{kibana-ref}/console-kibana.html[Kibana console] or command line. They are _not_
```

Comment:
Though this is taken from the existing https://www.elastic.co/guide/en/elasticsearch/reference/master/cat.html page, IMO (1) folks know what the Kibana console is without requiring a link and (2) the order of terms should be flipped, otherwise it sounds like there's a "Kibana command line":
```suggestion
command line or {kib} console. They are _not_
```